src/lib/Parser.hs

```diff
@@ -281,11 +281,8 @@ interfaceDef = do
   where
     mkFunDefs :: (SrcPos, Name, Name) -> UExpr -> [UDecl]
     mkFunDefs meta recordType
-      | (WithSrc _ (URecordTy (NoExt (LabeledItems items)))) <- recordType =
-          mkFunDefs' $ M.toList items
-      | otherwise = error "this should not be reachable"
-      where
-        mkFunDefs' = fmap (\(name, ty :| []) -> mkOneFunDef meta (name, ty))
+      | ~(WithSrc _ (URecordTy (NoExt (LabeledItems items)))) <- recordType =
```

Comment:
```suggestion
    mkFunDefs meta ~(WithSrc _ (URecordTy (NoExt (LabeledItems items)))) =
```

Reply:
(this is incorrect, because you also have to remove previous line, but that's what I meant in the original comment).